definitions/tools/cellranger_atac_count.cwl

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Run Cell Ranger ATAC Count"
+
+baseCommand: ["/opt/cellranger-atac-1.0.1/cellranger-atac", "count", "--localmem=64", "--localcores=8"]
```

Comment:
We could possibly set the `--localmem` and `--localcores` based on `runtime.cores` and `runtime.ram`; in some environments the request and how much is given can differ depending on available machine sizes.

Reply:
for converting the `runtime.ram` in mebibytes to gigabytes should I just get a close estimate by dividing by 953.7? or do something more exact like
```
runtime.ram * 1024^2 / 1000^3
```
and in general would it be good to allow a little overhead of requested ram to ram passed in as input to programs or trust them not to go over? if so divide by 960?

Reply:
It would certainly be good to leave a little bit of overhead. Something simple that leaves some room should be fine.

Reply:
done! I'll make another PR to do the same thing for the other cellranger tools